### PR TITLE
ArduCopter: Update descriptions of stream rate parameters

### PR DESCRIPTION
**Pull request by amilcarlucas**

### CODE REVIEW

File: ArduCopter/GCS_Mavlink.cpp [hunk 1]
```diff
@@ -627,7 +627,7 @@ bool GCS_MAVLINK_Copter::try_send_message(enum ap_message id)
 const AP_Param::GroupInfo GCS_MAVLINK::var_info[] = {
     // @Param: RAW_SENS
     // @DisplayName: Raw sensor stream rate
-    // @Description: Stream rate of RAW_IMU, SCALED_IMU2, SCALED_PRESSURE, and SENSOR_OFFSETS to ground station
+    // @Description: Stream rate of IMU1 (RAW_IMU, RAW_COMPASS), IMU2 (SCALED_PRESSURE) and IMU3 (SENSOR_OFFSETS) to ground station
```
Comment (khancyr) [inline ArduCopter/GCS_Mavlink.cpp]: using IMU X make the description weird. It should be better to fix the message.

File: ArduCopter/GCS_Mavlink.cpp [hunk 2]
```diff
@@ -636,7 +636,7 @@ const AP_Param::GroupInfo GCS_MAVLINK::var_info[] = {
 
     // @Param: EXT_STAT
     // @DisplayName: Extended status stream rate to ground station
-    // @Description: Stream rate of SYS_STATUS, MEMINFO, MISSION_CURRENT, GPS_RAW_INT, NAV_CONTROLLER_OUTPUT, and LIMITS_STATUS to ground station
+    // @Description: Stream rate of SYS_STATUS, POWER_STATUS, MEMINFO, CURRENT_WAYPOINT, GPS_RAW_INT, NAV_CONTROLLER_OUTPUT, and LIMITS_STATUS to ground station
```
Comment (OXINARF) [inline ArduCopter/GCS_Mavlink.cpp]: This looks correct, although the last one is, already, incorrect: it actually sends FENCE_STATUS.

File: ArduCopter/GCS_Mavlink.cpp [hunk 3]
```diff
@@ -690,7 +690,7 @@ const AP_Param::GroupInfo GCS_MAVLINK::var_info[] = {
 
     // @Param: EXTRA3
     // @DisplayName: Extra data type 3 stream rate to ground station
-    // @Description: Stream rate of AHRS, HWSTATUS, and SYSTEM_TIME to ground station
+    // @Description: Stream rate of AHRS, HWSTATUS, SYSTEM_TIME, RANGEFINDER, RANGEFINDER, TERRAIN, BATTERY2, MOUNT_STATUS, OPTICAL_FLOW, GIMBAL_REPORT, MAG_CAL_REPORT, MAG_CAL_PROGRESS, EKF_STATUS_REPORT, VIBRATION and RPM to ground station
```
Comment (OXINARF) [inline ArduCopter/GCS_Mavlink.cpp]: RANGEFINDER twice. Second one should be DISTANCE_SENSOR.

TERRAIN -> TERRAIN_REQUEST

File: ArduCopter/GCS_Mavlink.cpp [hunk 4]
```diff
@@ -750,16 +750,16 @@ GCS_MAVLINK_Copter::data_stream_send(void)
     }
 
     if (stream_trigger(STREAM_RAW_SENSORS)) {
-        send_message(MSG_RAW_IMU1);
-        send_message(MSG_RAW_IMU2);
-        send_message(MSG_RAW_IMU3);
+        send_message(MSG_RAW_IMU1);  // RAW_IMU, RAW_COMPASS
```
Comment (OXINARF) [inline ArduCopter/GCS_Mavlink.cpp]: Like I wrote in the other comment, no RAW_COMPASS, and SCALED_IMU2 should be here too.

File: ArduCopter/GCS_Mavlink.cpp (hunk 1 above)
Comment (amilcarlucas) [inline ArduCopter/GCS_Mavlink.cpp]: Thanks, I will rework this on monday and re-submit

**Comment by amilcarlucas** [comment]
Requested changes have been done

**Comment by OXINARF** [comment]
Squashed, added missing AHRS2 and added a commit on top to fix LIMITS_STATUS to FENCE_STATUS. Thanks!

**Pull request closed without merging**